python/dgl/transform.py

```diff
@@ -613,14 +673,16 @@ def metis_partition_assignment(g, k):
         node_part = utils.toindex(node_part)
         return node_part.tousertensor()
 
-def metis_partition(g, k, extra_cached_hops=0):
+def metis_partition(g, k, extra_cached_hops=0, reshuffle=False):
```

Comment:
Can we move the reshuffle(remapping) task to a standalone func? So we do not need to implement it in each partition algorithm.

Reply:
reshuffling happens in `partition_graph_with_halo`. it works for different partitioning algorithms.